### PR TITLE
adding docs for createLink and Invite API changes

### PR DESCRIPTION
**Pull request by agarwal-arjun**
adding docs for createLink and Invite API changes

### CODE REVIEW

File: api-reference/beta/api/driveitem-createlink.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ The request should be a JSON object with the following properties.
 |expirationDateTime|DateTimeOffset|Optional. A String with format of yyyy-MM-ddTHH:mm:ssZ of DateTime indicates the expiration time of the permission.|
 |password|String|Optional.The password of the sharing link that is set by the creator.|
 |recipients|[driveRecipient](../resources/driverecipient.md) collection|Optional. A collection of recipients who will receive access to the sharing link.|
+| retainInheritedPermissions |  Boolean          | Optional. If `true` (default), any current existing inherited permissions are retained on the shared item when sharing this item for the first time. If `false`, all existing permissions are removed when sharing for the first time.  |
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-createlink.md:61]: Global: To reduce wordiness, I suggest saying current or existing, but not both, as both words convey the same meaning.

File: api-reference/v1.0/api/driveitem-createlink.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ The request should be a JSON object with the following properties.
 | **password** | string | The password of the sharing link that is set by the creator. Optional and OneDrive Personal only.
 | **expirationDateTime** | string | A String with format of yyyy-MM-ddTHH:mm:ssZ of DateTime indicates the expiration time of the permission. |
 | **scope** | string | Optional. The scope of link to create. Either `anonymous` or `organization`. |
+| **retainInheritedPermissions** |  boolean          | Optional. If `true` (default), any current existing inherited permissions are retained on the shared item when sharing this item for the first time. If `false`, all existing permissions are removed when sharing for the first time.  |
```
Comment (Lauragra) [inline api-reference/v1.0/api/driveitem-createlink.md:54]: Please capitalize Boolean.

File: changelog/Microsoft.FileServices.json [hunk 3]
```diff
@@ -1,5 +1,95 @@
 {
   "changelog": [
+    {
+      "ChangeList":[
+        {
+          "Id": "FA782B9D-526C-403F-AA57-8144A4E736FB",
+          "ApiChange": "Parameter",
+          "ChangedApiName": "retainInheritedPermissions",
+          "ChangeType": "Addition",
+          "Description": "Added the **retainInheritedPermissions** (string) property to the [createLink](https://docs.microsoft.com/en-us/graph/api/driveitem-createlink?view=graph-rest-beta) action on a [driveItem](https://docs.microsoft.com/en-us/graph/api/resources/driveitem?view=graph-rest-beta) resource.",
```
Comment (Lauragra) [inline changelog/Microsoft.FileServices.json:10]: Global: Should you refer to retainInheritedPermissions as a property or a parameter?

File: api-reference/beta/api/driveitem-createlink.md (hunk 1 above)
Comment (agarwal-arjun) [reply api-reference/beta/api/driveitem-createlink.md]: addressed

File: api-reference/v1.0/api/driveitem-createlink.md (hunk 2 above)
Comment (agarwal-arjun) [reply api-reference/v1.0/api/driveitem-createlink.md]: addressed.

File: changelog/Microsoft.FileServices.json (hunk 3 above)
Comment (agarwal-arjun) [reply changelog/Microsoft.FileServices.json]: Changed it to "parameter".

**Review by Lauragra — APPROVED** [review]

**Comment by Lauragra** [comment]
@agarwal-arjun , are we ready to publish this PR? The schema has already been released. Thanks.

**Comment by agarwal-arjun** [comment]
Not yet. I will update here once we are ready.

**Comment by Lauragra** [comment]
> Not yet. I will update here once we are ready.

@agarwal-arjun , do you have an update on when you would like to publish this PR? Thanks.

**Comment by agarwal-arjun** [comment]
@Lauragra, I expect to publish in ballpark of next 1-2 month.

**Comment by Lauragra** [comment]
@agarwal-arjun , are we ready yet to publish this PR?

**Comment by agarwal-arjun** [comment]
> 

This is taking more time than usual. We are gradually rolling out the flight and it will take bit of more time to complete. Until then we will need to wait to publish this.

**Pull request merged**